07-hosting.Rmd

```diff
@@ -123,9 +123,9 @@ Then you'd be able to use `docker run --rm -v `pwd`:/app myCustomDocker` to run
 
 ### Automatically Run on Restart
 
-If you want your container to start automatically when your machine is booted, you can use the `-d` switch for `docker run`.
+If you want your container to start automatically when your machine is booted, you can use the `--restart` parameter for `docker run`.
```

Comment:
Is this specific to a particular version of docker? I'm not sure what benefits, if any, this offers given the PR context here.